vipps-login-api-quick-start.md

```diff
@@ -100,10 +104,12 @@ curl https://apitest.vipps.no/access-management-1.0/access/.well-known/openid-co
 </TabItem>
 </Tabs>
 
+The URL of the OP's OAuth 2.0 Authorization Endpoint is provided in the response.
```

Comment:
I couldn't see that OP is defined anywhere. I assume it refers to OpenID provider from the OIDC standard?

```suggestion
The URL of the OP's OAuth 2.0 Authorization Endpoint is provided in the response.
```

Reply:
Yes, I copied this from the swagger file and it uses "OpenID Provider" there.